cea/demand/thermal_loads.py

```diff
@@ -1205,19 +1211,24 @@ def get_temperatures(locator, prop_HVAC):
     prop_emission_heating = pd.read_excel(locator.get_technical_emission_systems(), 'heating')
     prop_emission_cooling = pd.read_excel(locator.get_technical_emission_systems(), 'cooling')
     prop_emission_dhw = pd.read_excel(locator.get_technical_emission_systems(), 'dhw')
+    prop_system_vent = pd.read_excel(locator.get_technical_emission_systems(), 'ventilation')
+    prop_system_vent_ctrl = pd.read_excel(locator.get_technical_emission_systems(), 'ventilation_control')
 
     df_emission_heating = prop_HVAC.merge(prop_emission_heating, left_on='type_hs', right_on='code')
     df_emission_cooling = prop_HVAC.merge(prop_emission_cooling, left_on='type_cs', right_on='code')
     df_emission_dhw = prop_HVAC.merge(prop_emission_dhw, left_on='type_dhw', right_on='code')
+    df_system_ctrl_vent = prop_system_vent.merge(prop_system_vent_ctrl,left_on='code_ctrl', right_on='code', suffixes={'_v','_c'})
+    df_system_ctrl_vent = prop_HVAC.merge(df_system_ctrl_vent, left_on='type_vent', right_on='code_v')
 
-    fields_emission_heating = ['Name', 'type_hs', 'type_cs', 'type_dhw', 'type_ctrl', 'Tshs0_C', 'dThs0_C',
-                               'Qhsmax_Wm2']
+
+    fields_emission_heating = ['Name', 'type_hs', 'type_cs', 'type_dhw', 'type_ctrl', 'Tshs0_C', 'dThs0_C', 'Qhsmax_Wm2']
     fields_emission_cooling = ['Name', 'Tscs0_C', 'dTcs0_C', 'Qcsmax_Wm2']
     fields_emission_dhw = ['Name', 'Tsww0_C', 'dTww0_C', 'Qwwmax_Wm2']
+    fields_system_ctrl_vent = ['Name', 'MECH_VENT', 'WIN_VENT', 'HEAT_REC', 'NIGHT_FLSH', 'ECONOMIZER']
 
     result = df_emission_heating[fields_emission_heating].merge(df_emission_cooling[fields_emission_cooling],
```

Comment:
this is getting tedious. maybe we should create separate statements for each merge...